### ISSUE TITLE
Self-Service - fails to configure properly with a TimeSpan overflow exception when commandExecutionTimeout is set 0 (infinite)

### ISSUE THREAD

**Issue by steviecoaster** [chocolatey/chocolatey-licensed-issues#172]
### What You Are Seeing?

Background service doesn't work and fails with an error about `TimeSpan overflowed because the duration is too long`.

### What is Expected?

Background service should configure itself properly and work with an infinite timeout.

### How Did You Get This To Happen? (Steps to Reproduce)

Set commandExecutionTimeout to 0 and attempt to install software using the background agent

### Output Log

```sh
2020-08-28 12:20:58,399 9384 [DEBUG] - [Background] Issue when attempting background service connection:
 System.OverflowException: TimeSpan overflowed because the duration is too long.
   at System.TimeSpan.Interval(Double value, Int32 scale)
   at chocolatey.licensed.infrastructure.app.utility.BackgroundServiceUtility.configure_background_service(LicensedChocolateyConfiguration configuration)
2020-08-28 12:20:58,401 9384 [WARN ] - [Background] Chocolatey is configured for background service, but there is no 
 background service available and running. Please ensure you have 
 installed the service - 'choco install chocolatey-agent' (or upgraded 
 to at least 0.5.0 - earlier versions are not compatible). If you have
 mistakenly turned on the background service, please run 
 'choco feature disable -n useBackgroundService'.
 TimeSpan overflowed because the duration is too long.
```

### References

[Internal Issue](https://gitlab.com/chocolatey/collaborators/choco-licensed/-/issues/346)

**Comment by steviecoaster** [comment]
Workaround is to run `choco config set commandExecutionTimeout 108000` which should make the timeout be 30 hours.

**Comment by ryanrichter94** [comment]
Client confirmed this as a valid workaround for the issue.

**Comment by ferventcoder** [comment]
This has been fixed for Chocolatey Licensed Extension v2.1.2
